src/test/resources/templates/multi-fragment.html

```diff
@@ -0,0 +1,2 @@
+<div id="thing" hx-swap-oob="true" th:text="${thing}"></div>
+<div th:include="~{__${fragmentName}__}" ></div>
```

Comment:
Realistically this second one would be a repeated of `oob-swap` elements that we could add to (or leave empty when not needed)